op-node/rollup/derive/calldata_source.go

```diff
@@ -173,10 +172,10 @@ func DataFromEVMTransactionsV2(config *rollup.Config, txs types.Transactions, re
 			receipt := receipts[j]
 			data := tx.Data()
 			// Exclude transactions if L1 transaction did call submit function.
```

Comment:
The comment here should say `Exclude transactions if L1 transaction did not call submit function.` Currently, the "not" is missing.